blog_on_rails/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,82 @@
+class UsersController < ApplicationController
+    
+    before_action :find_user
+    def new
+        @user = User.new
+    end
+
+    def create
+        @user = User.new user_params
+        if @user.save
+            session[:user_id] = @user.id
+            redirect_to root_path
+        else
+            render :new
+        end
+    end
+
+    def edit
+        
+    end
+
+    def update
```

Comment:
should only allow the correct signed in user access to this view

also, should validate that the correct user is requesting changes before changes can be written to the database